packages/destructure/src/index.ts

```diff
@@ -4,30 +4,34 @@ import { access, MaybeAccessor, AnyObject, Values, AnyFunction } from "@solid-pr
 type ReactiveSource = [] | any[] | AnyObject;
 
 export type DestructureOptions<T extends ReactiveSource> = MemoOptions<Values<T>> & {
-  memo?: boolean;
+  memo?: boolean | "normalize";
```

Comment:
Yeah I don't like the idea of combining the two. Normalize is about the inputs, while memo is about the outputs. I don't see a point in limiting options to either choosing one or another. `{ memo: boolean, normalize: boolean }` is simpler and lets the user decide on the bahavior.

e.g. here I would like to both normalize the inputs, and memoize the outputs:
```ts
const [count, setCount] = createSignal(1)

const {foo, bar} = destructure({
	get foo() {return count() > 5},
	bar: () => count() > 5
}, {memo: true, normalize: true})

createEffect(() => {console.log("foo", foo())})
createEffect(() => {console.log("bar", bar())})

setCount(2) // shouldn't rerun effects
setCount(6) // should rerun effects
```

or rely on the original singals:

```ts
const [count, setCount] = createSignal(1)

const {foo, bar} = destructure({
	get foo() {return count() > 5},
	bar: () => count() > 5
}, {normalize: true})

createEffect(() => {console.log("foo", foo())})
createEffect(() => {console.log("bar", bar())})

setCount(2) // should rerun effects
setCount(6) // should rerun effects
```

Reply:
I added this this to the test.